python/pyarrow/tests/test_schema.py

```diff
@@ -564,8 +566,10 @@ def test_schema_from_pandas():
             '2007-07-13T01:23:34.123456789',
             '2006-01-13T12:34:56.432539784',
             '2010-08-13T05:46:57.437699912'
-        ], dtype='datetime64[ns]')
+        ], dtype='datetime64[ns]'),
     ]
+    if LooseVersion(pd.__version__) >= '0.24.0':
```

Comment:
`pa.array(..)` relies on `__arrow_array__` being defined on the array to be able to convert pandas nullable integer array, and that was only added in pandas 1.0 I think

Reply:
Thanks for the information, all green now.